runtimes/api.py

```
@@ -98,9 +106,34 @@ async def refresh_runtime_health() -> dict:
+        return store.load(_RICH_POLICY_KEY) or {}
+    except Exception:
+        return {}
+
+
+def _save_rich_policy(data: dict) -> None:
+    try:
+        from webui.config_store import JsonConfigStore
+        store = JsonConfigStore()
+        store.save(_RICH_POLICY_KEY, data)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)